### PR TITLE
feat: Update ShortKey for stop query running in SqlLab editor

### PR DESCRIPTION
**Pull request by codemaster08240328**
### SUMMARY

- Change the shortcut for Stop Query from Control + X to Control + E on PC / Linux
- Update the tooltip text on the stop query button to reflect the shortcut change
- Mac shortcut should remain the same as it is today (Control+X)

### TESTING INSTRUCTIONS

1. Go to SqlLab
2. Run any query
3. Try to press ctrl + e(on PC, Linux) or ctrl + x (on Mac)
4. Check tool tip when hovering on `Stop query` button

### ADDITIONAL INFORMATION

- [x] Has associated issue: https://github.com/apache/superset/issues/9990

### CODE REVIEW

File: superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx [hunk 1]
```diff
@@ -114,7 +117,9 @@ const RunQueryActionButton = ({
         tooltip={
           (!isDisabled &&
             (shouldShowStopBtn
-              ? t('Stop running (Ctrl + x)')
+              ? userOS === 'MacOS'
```
Comment (diegomedina248) [inline superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx:120]: we could move this to a memo, or even a plain function outside of the component called `getStopButtonTooltipText` or something.. Seems too much inline ifs for my liking here

Comment (rusackas) [reply superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx]: Willing to take a look at externalizing this, and squeeze it into this PR, @codemaster08240328?

File: superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ import {
   DropdownButtonProps,
 } from 'src/components/DropdownButton';
 import { detectOS } from 'src/utils/common';
+import { useMemo } from '@storybook/addons';
```
Comment (diegomedina248) [inline superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx:30]: This one should be imported from React

Comment (rusackas) [reply superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx]: Correct... and we can add this (and the lodash one) as an eslint restricted-imports rule.

**Review by rusackas — APPROVED** [review]

**Comment by rusackas** [comment]
Tests for this would be appreciated as a follow-up PR.

**Pull request merged**